src/picongpu/include/fields/MaxwellSolver/Lehe/LeheCurl.hpp

```diff
@@ -42,53 +42,62 @@ namespace picongpu
             typedef PMacc::math::CT::Int< 1, 1, 1 > LowerMargin;
             typedef PMacc::math::CT::Int< 2, 2, 2 > UpperMargin;
 
+            float_X mySin;
+
+            HINLINE CurlELehe( )
```

Comment:
why are the two implementations of `CurlELehe( )`?
here and line 190 of **this file**